core/templates/dev/head/pages/exploration_editor/editor_tab/ResponsesService.js

```diff
@@ -160,10 +125,6 @@ oppia.factory('ResponsesService', [
         newConfirmedUnclassifiedAnswers, oldConfirmedUnclassifiedAnswers)) {
         _confirmedUnclassifiedAnswers = newConfirmedUnclassifiedAnswers;
 
-        ExplorationStatesService.saveConfirmedUnclassifiedAnswers(
```

Comment:
Hm... where's the replacement for this?

Reply:
They have been added to TrainingDataService from where these functions were called.